### PR TITLE
Reorganization of the code base

### PR DESCRIPTION
**Pull request by Savichev-Igor**
I tried to follow this [pattern](https://github.com/golang-standards/project-layout/blob/master/README.md)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/brotli"]
```
Comment (Tinsane) [inline .gitmodules]: How did you solve the problem with brotli internal linking?

File: Gopkg.toml [hunk 2]
```diff
@@ -1,26 +1,86 @@
+# Gopkg.toml example
```
Comment (Tinsane) [inline Gopkg.toml]: Is there any need to leave this example here as it is?

Comment (Savichev-Igor) [reply Gopkg.toml]: It's autogenerated file from [dep](https://github.com/golang/dep)

With his help we can remove `vendors/` from repo =)

File: .gitmodules (hunk 1 above)
Comment (Savichev-Igor) [reply .gitmodules]: I'm working on it, but now i just move `.brotli` to `git submodule`

In my opinion is a more correct solution and expandable

**Pull request merged**